packages/create-osdk-app/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "create-osdk-app",
```

Comment:
should this have the `@osdk/` prefix? or is that handled somewhere else?

Reply:
(I think that it has to be named this in order for us to match the other `npx create-foo-app` syntax floating around the JS ecosystem)

One thing we want to do though is make sure the NPM token has permissions to publish to this package as well.